website/blog/2020-09/_index.md

```diff
@@ -0,0 +1,80 @@
+Whilst we continue to work on topics related to scalability, robustness, and better observability, we agreed to adjust our focus a little more into the areas of development productivity, code quality and unit/integration testing for the upcoming releases.
+
+## Notable Changes in v1.10
+
+[Gardener v1.10](https://github.com/gardener/gardener/releases/tag/v1.10.0) was a comparatively small release (measured by the number of changes) but it comes with some major features!
+
+### Kubernetes 1.19 support ([gardener/gardener#2799](https://github.com/gardener/gardener/pull/2799))
+
+The newest minor release of Kubernetes is now supported by Gardener (and all the maintained provider extensions)!
+Predominantly, we have enabled CSI migration for OpenStack now that it got promoted to beta, i.e., 1.19 shoots will no longer use the in-tree Cinder volume provisioner.
+The CSI migration enablement for Azure got postponed (to at least 1.20) due to some issues that the Kubernetes community is trying to fix in the 1.20 release cycle.
+As usual, the [1.19 release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) should be considered before upgrading your shoot clusters.
+
+### Automated certificate rotation for gardenlet ([gardener/gardener#2542](https://github.com/gardener/gardener/pull/2542))
+
+Similar to the kubelet, the gardenlet supports TLS bootstrapping when deployed into a new seed cluster.
+It will request a client certificate for the garden cluster using the `CertificateSigningRequest` API of Kubernetes and store the generated results in a `Secret` object in the `garden` namespace of its seed.
+These certificates are usually valid for one year.
+We have now added support for automatic renewals if the expiration dates are approaching.
+
+### Improved monitoring alerts ([gardener/gardener#2776](https://github.com/gardener/gardener/pull/2776))
+
+We have worked on a larger refactoring to improve reliability and accuracy of our monitoring alerts for both shoot control planes in the seed as well as shoot system components running on worker nodes.
+The improvements are primarily for operators and should result in less false positive alerts.
+Also, the alerts should fire less frequently and are better grouped in order to reduce to overall amount of alerts.
+
+### Seed deletion protection ([gardener/gardener#2732](https://github.com/gardener/gardener/pull/2732))
+
+Our validation to improve robustness and countermeasures against accidental mistakes has been improved.
+Earlier, it was possible to remove the `use-as-seed` annotation for [shooted seeds](https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md) or directly set the `deletionTimestamp` on `Seed` objects despite of the fact that they might still run shoot control planes.
+Seed deletion would not start in these cases, though, it would disrupt the system unnecessarily, and result in some unexpected behaviour.
```

Comment:
```suggestion
Seed deletion would not start in these cases, although it would disrupt the system unnecessarily, and result in some unexpected behaviour.
```